wagtail_ab_testing/static_src/main.tsx

```diff
@@ -1,5 +1,36 @@
+import c3 from 'c3';
```

Comment:
I just learned about c3, nice!

Reply:
Yep, shamelessly pinched from wagtail-experiments! (well, I did convert it to use the npm package)